### ISSUE TITLE
Find case examples of large accessible hideable navigation

### ISSUE THREAD

**Issue by bengolder** [codeforamerica/intake#1127]
Criteria:
- should be easily navigable using voiceover or keyboard
- should be able to "slide" away the navigation to hide it
- should work on mobile

**Comment by bengolder** [comment]
sorry, wrong repo. closing.
